CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
+
+IMPROVEMENTS:
+* alloydb: changed the resource `google_alloydb_instance` to be created directly with public IP enabled instead of creating the resource with public IP disabled and then enabling it ([#7469](https://github.com/hashicorp/terraform-provider-google-beta/pull/7469))
+* bigtable: added `automated_backup_configuration` field to `google_bigtable_table` resource ([#7468](https://github.com/hashicorp/terraform-provider-google-beta/pull/7468))
+* cloudbuildv2: added support for connecting to Bitbucket Data Center and Bitbucket Cloud with the `bitbucket_data_center_config` and `bitbucket_cloud_config` fields in `google_cloudbuildv2_connection` ([#7494](https://github.com/hashicorp/terraform-provider-google-beta/pull/7494))
+* compute: added support for Port Mapping NEG and endpoint. New NEG type is a regional type of `GCE_VM_IP_PORTMAP` that requires endpoints with `instance`, `port` and `client_destination_port` (beta) ([#7471](https://github.com/hashicorp/terraform-provider-google-beta/pull/7471))
+* compute: added update support to 'ssl_policy' field in 'compute_region_target_https_proxy' resource ([#7484](https://github.com/hashicorp/terraform-provider-google-beta/pull/7484))
+* compute: removed enum validation on `guest_os_features.type` in `google_compute_disk` to allow for new features to be used without provider update ([#7465](https://github.com/hashicorp/terraform-provider-google-beta/pull/7465))
+* compute: updated documentation of google_compute_target_https_proxy and google_compute_region_target_https_proxy ([#7481](https://github.com/hashicorp/terraform-provider-google-beta/pull/7481))
+* container: added support for `security_posture_config.mode` value "ENTERPRISE" in `resource_container_cluster` ([#7467](https://github.com/hashicorp/terraform-provider-google-beta/pull/7467))
+* discoveryengine: added `document_processing_config` field to `google_discovery_engine_data_store` resource ([#7475](https://github.com/hashicorp/terraform-provider-google-beta/pull/7475))
+* edgecontainer: added 'maintenanceExclusions' field to 'edgecontainer_cluster' resource ([#7490](https://github.com/hashicorp/terraform-provider-google-beta/pull/7490))
+* gkehub: added `prevent_drift` field to ConfigManagement `fleet_default_member_config` ([#7464](https://github.com/hashicorp/terraform-provider-google-beta/pull/7464))
+* netapp: added `administrators` field to `google_netapp_active_directory` resource ([#7466](https://github.com/hashicorp/terraform-provider-google-beta/pull/7466))
+* networksecurity: added field update tests for `google_network_securtiy_tls_inspection_policy` ([#7491](https://github.com/hashicorp/terraform-provider-google-beta/pull/7491))
+* vertexai: promoted `optimized` field to GA for `google_vertex_ai_feature_online_store` resource ([#7473](https://github.com/hashicorp/terraform-provider-google-beta/pull/7473))
```

Comment:
Are promotions to GA included in Beta changelogs?

Reply:
I've seen this type of entries in past changelogs, will leave this here unless there's strong opinion to remove this line.